docs/pages/Nakama/plugin.mdx

```diff
@@ -0,0 +1,201 @@
+# Cardinal Plugin
+
+Go based Nakama containers can be extended using shared objects (aka [plugins](https://pkg.go.dev/plugin)). The [starter-game-template](https://github.com/Argus-Labs/starter-game-template) leverages this shared object pattern to expand on Nakama to better interact with a Cardinal based backend.
+
+The Nakama plugin bundled with the starter-game-template is general purpose, and should work with your Cardinal based game without any modifications. As you work on your game, you may find that you want to further change the Nakama plugin to better serve your specific use case. This document serves as a reference for the various aspects of the Cardinal-focused Nakama plugin.
+
+## RPC Endpoints
+
+Transactions and Reads registered with a `cardinal.World` object will automatically get Nakama RPC endpoints created for them. For example, if you set up a Transaction and a Read with the following code:
+
+```go
+var MoveTx = cardinal.NewTransactionType[MoveMsg, MoveResult]("move-a-player")
+var PositionRead = cardinal.NewReadType[PositionRequest, PositionReply]("position")
```

Comment:
maybe this is not important, but there aren't enough args to `NewReadType` here. maybe we can ellipsis to indicate there should be more after "position"

Reply:
I added a placeholder function name that isn't actually defined anywhere. Let me know if you think this should be "..." instead (or maybe a stubbed anonymous function).